app/styles/ember-appuniversum/_c-link.scss

```diff
@@ -43,13 +41,6 @@ $au-link-secondary-active-color          : var(--au-gray-900) !default;
     outline: var(--au-outline);
     outline-offset: var(--au-outline-offset);
   }
-
-  &.is-active {
```

Comment:
I don't think these styles should be removed? It affects the `AuLink` component as well and there it does make sense.

Reply:
Oh! I was a little fast there. I'll revert those changes!